src/components/switchTheme/useSwitchTheme.test.tsx

```diff
@@ -1,42 +1,27 @@
-import { act } from 'react';
-import { renderHook } from '@testing-library/react';
+import { act, renderHook } from '@testing-library/react';
 import { describe, expect, it, vi } from 'vitest';
-import { THEMES, useSwitchTheme } from 'src/components/switchTheme/hook';
-import { useDispatch, useSelector } from 'react-redux';
-import { toggleTheme } from 'src/store/slices/switchTheme';
+import { ThemeContextProvider } from 'src/components/switchTheme/useThemeContext';
+import { useTheme } from 'src/components/switchTheme/hook';
 
-vi.mock('react-redux', () => ({
-  useDispatch: vi.fn(),
-  useSelector: vi.fn(),
-}));
+describe('useTheme Hook', () => {
+  it('should toggle theme correctly', () => {
```

Comment:
tho toggle it correctly it should have initial values. "black theme" and "white theme" in the local storage :)

Reply:
done